src/cli/index.js

```diff
@@ -382,7 +343,7 @@ config.init({
   httpsProxy: commander.httpsProxy,
   networkConcurrency: commander.networkConcurrency,
   nonInteractive: commander.nonInteractive,
-  commandName,
+  commandName: commandName === 'run' ? commander.args[0] : commandName,
```

Comment:
This eventing I have to remember to open at least one issue, otherwise I will forget everything :)